ConcurrentWitness2Test/start.sh svcomp23-witness-benchmarks/tools/infer/goblint-regression/13-privatized_67-pthread_cond_wait_unknown_1_neg.yml/13-privatized_67-pthread_cond_wait_unknown_1_neg.i --graphml-witness svcomp23-witness-benchmarks/tools/infer/goblint-regression/13-privatized_67-pthread_cond_wait_unknown_1_neg.yml/witness.graphml


--------------------------------------------------------------------------------


/home/runner/work/ConcurrentWitness2Test/ConcurrentWitness2Test/ConcurrentWitness2Test/venv/lib/python3.12/site-packages/networkx/readwrite/graphml.py:1034: UserWarning: No key type for id creationtime. Using string
  warnings.warn(f"No key type for id {attr_id}. Using string")
Traceback (most recent call last):
  File "/home/runner/work/ConcurrentWitness2Test/ConcurrentWitness2Test/ConcurrentWitness2Test/main.py", line 57, in translate_to_c
    reach_error(ast)
  File "/home/runner/work/ConcurrentWitness2Test/ConcurrentWitness2Test/ConcurrentWitness2Test/tweaks.py", line 24, in reach_error
    extern_decl = Decl(
                  ^^^^^
TypeError: Decl.__init__() missing 1 required positional argument: 'align'
Verdict: Unknown error
